adal/tests/Test.ADAL.NET.UIAutomation/CoreMobileADALTests.cs

```diff
@@ -37,37 +40,131 @@ namespace Test.ADAL.NET.UIAutomation
 	public class CoreMobileADALTests
     {
         /// <summary>
-        /// Runs through the standard acquire token flow
+        /// Runs through the standard acquire token interactive flow
         /// </summary>
         /// <param name="controller">The test framework that will execute the test interaction</param>
-        public static void AcquireTokenTest(ITestController controller)
+        public static void AcquireTokenInteractiveTest(ITestController controller, UserQueryParameters userParams)
 		{
-            //Get User from Lab
-            var user = controller.GetUser(
-                new UserQueryParameters
-                {
-                    IsMamUser = false,
-                    IsMfaUser = false,
-                    IsFederatedUser = false
-                });
+            AcquireTokenInteractivly(controller, userParams);
+            VerifyResult(controller);
+        }
+
+        /// <summary>
+        /// Runs through the standard acquire token silent flow
+        /// </summary>
+        /// <param name="controller">The test framework that will execute the test interaction</param>
+        public static void AcquireTokenSilentTest(ITestController controller, UserQueryParameters userParams)
```

Comment:
All the methods in this class are static and all take in a test controller and user params. This is a code smell that in fact this class should not be static and ITestController is a member of the class. Probably UserQueryParameters as well.

Reply:
What would be the benefit of making the class concrete? Also, if I make ITestController a member of the class, each test that uses this class would need to first set the ITestController before the test code is executed anyway. Thus, I thought it was better to simply pass the ITestController directly into the function that contains the test code, saving a few lines of code overall.

Reply:
This is meant to be more of a helper class for the automation tests. I have renamed it so that it makes more sense. Please see my changes latest. Perhaps it will change your opinion?